src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2PipelineOptions.java

```diff
@@ -121,6 +121,16 @@
   int getNumShardsPerDLPRequestBatching();
   void setNumShardsPerDLPRequestBatching(int value);
 
+  @Description("Number of retries in case of transient errors in dlp api")
+  @Default.Integer(10)
+  int getDlpApiRetryCount();
+  void setDlpApiRetryCount(int value);
+
+  @Description("Initial backoff (in seconds) for retries with exponential backoff")
+  @Default.Integer(5)
+  int getInitialBackoff();
```

Comment:
Some Java doc comments on how these values are used would help users understand what to expect.